create-addons-loader.js

```diff
@@ -68,6 +74,14 @@ const safeWrapper = (func) => (config) => {
   return res;
 }
 
+const voltoConfFileConfig = (config) => {
```

Comment:
@tiberiuichim Also implemented the `volto.conf.js` thingy. What do you think?

Reply:
whohooo!